desktop/apps/home/routes.coffee

```diff
@@ -59,7 +59,7 @@ fetchMetaphysicsData = (req)->
   }
 
   Q
-    .all [
+    .allSettled [
```

Comment:
Since we're not adding handling the possibility of the _other_ requests below failing (`fetchMetaphysicsData`, `featuredLinks.fetch`, `featuredShows.fetch`), I assume we're comfortable 500-ing in those cases? I'm not sure how critical they are to the page.

Reply:
Also: how would this handle the case of Positron just responding very slowly? Would Force's home page rendering wait until some ~30 second time-out, or are there tighter constraints imposed on these requests somehow? (Might make sense if possible.)

Reply:
This will handle all of the requests failing in that block. Since `fetchMetaphysicsData` is pretty crucial for the page tho, it will likely 500 from a rendering error but for the others it just won't render its contents. (Actually let me update some of the templates to better reflect that).

As for timeouts, I want to try and replicate those to see what happens. I _think_ it would just 500 the page -- my guess is that the 30-second timeout is a more general middleware-level config. We can be more granular though. https://github.com/kriskowal/q/wiki/API-Reference#promisetimeoutms-message

Reply:
Sounds good. Let me know when those templates are updated as you mention.

Re. timeouts, I think this _would_ be a good candidate for tighter timeouts (esp. for non-essential requests once the templates can handle the absence of those responses), but that can be separate.